Tests/TuistGeneratorTests/Linter/Mocks/MockXcodeProjWriter.swift

```diff
@@ -0,0 +1,15 @@
+
+import Foundation
+@testable import TuistGenerator
+
+class MockXcodeProjWriter: XcodeProjWriting {
```

Comment:
If the answer to my comment about whether  `XcodeProjWriter` should be public is yes, I'd move this one to `TuistGeneratorTesting`.